client/dashboard/sites/overview-plan-card/index.tsx

```diff
@@ -1,22 +1,107 @@
+import { JetpackLogo } from '@automattic/components/src/logos/jetpack-logo';
 import { useQuery } from '@tanstack/react-query';
-import { __experimentalVStack as VStack } from '@wordpress/components';
+import {
+	__experimentalGrid as Grid,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+	Icon,
+	Tooltip,
+} from '@wordpress/components';
 import { cloneElement } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
-import { wordpress } from '@wordpress/icons';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	chartBar,
+	cloud,
+	megaphone,
+	next,
+	removeBug,
+	shield,
+	search,
+	video,
+	wordpress,
+} from '@wordpress/icons';
 import filesize from 'filesize';
 import { siteMediaStorageQuery } from '../../app/queries/site-media-storage';
 import { siteMetricsQuery } from '../../app/queries/site-metrics';
 import { siteCurrentPlanQuery } from '../../app/queries/site-plans';
 import { sitePurchaseQuery } from '../../app/queries/site-purchases';
 import { Stat } from '../../components/stat';
 import { DotcomPlans } from '../../data/constants';
+import { hasPlanFeature } from '../../utils/site-features';
+import { isSelfHostedJetpackConnected } from '../../utils/site-types';
 import { getSiteDisplayUrl } from '../../utils/site-url';
 import OverviewCard from '../overview-card';
+import type { JetpackModules } from '../../data/constants';
 import type { Site, Plan, Purchase } from '../../data/types';
 
+import './style.scss';
+
 const MINIMUM_DISPLAYED_USAGE = 2.5;
 const ALERT_PERCENT = 80;
 
+const JETPACK_MODULES = [
```

Comment:
Nit: `JETPACK_PRODUCTS`? I think it all makes sense now, since the referenced code is from `my-jetpack/src/"products"/` 😄 

Also `module` => `product`; `getJetpack{Module -> Product}sDescription()`, etc.

Reply:
Wait now I am confused whether they are "features" or "products" 😄 anyway, it should not be modules...

Reply:
A Jetpack product (e.g. Social), can have many features. One of those features (JETPACK_FEATURE introduced in this PR) is used to determine whether the site has the product or not.